HarryPotter/Screens/House/ViewModel/HouseViewModel.swift

```diff
@@ -29,15 +46,15 @@ protocol HouseViewModelProtocol {
     var delegate: HouseViewModelDelegate? { get set }
     
     func downloadCharacters()
-
 }
 
 final class HouseViewModel: HouseViewModelProtocol {
+
+    private let house: HouseOfHogwart
+    weak var delegate: HouseViewModelDelegate?
+    private let apiClient = APIClient()
     
-    private let house: House
-    var delegate: HouseViewModelDelegate?
-    
-    init(house: House) {
+    init(house: HouseOfHogwart) {
```

Comment:
you can inject `APIClient` here (dependency injection), so that we could mock it later in tests ;)